resources/views/components/checkboxes.blade.php

```diff
@@ -1,6 +1,10 @@
-@foreach($options as $value => $label)
+@foreach($options as $value => $atts)
 <div class="field">
-    <input {!! $attributes !!} type="checkbox" name="{{ $name }}[]" id="{{ $name }}-{{ $value }}" value="{{ $value }}" {!! $describedBy() ? 'aria-describedby="' . $describedBy() . '"' : '' !!} @if (in_array($value, $selected, true)) checked @endif {!! $invalid ? 'aria-invalid="true"' : '' !!} />
-    <x-hearth-label for="{{ $name }}-{{ $value }}">{{ $label }}</x-hearth-label>
+    @php($hint = isset($atts['hint']) ? $name . '-' . $value . '-hint' : '')
+    <input {!! $attributes !!} type="checkbox" name="{{ $name }}[]" id="{{ $name }}-{{ $value }}" value="{{ $value }}" {!! $describedBy($hint) ? 'aria-describedby="' . $describedBy($hint) . '"' : '' !!} @if (in_array($value, $selected, true)) checked @endif {!! $invalid ? 'aria-invalid="true"' : '' !!} />
```

Comment:
What does this do? `{{ $name }}[]`

Reply:
Adding the square brackets submits the values of this group of checkboxes as an array for PHP. Here's an example: https://html.form.guide/php-form/php-form-checkbox/

Reply:
Thanks that's helpful.